### ISSUE TITLE
 yaml.load() deprecation warning: YAMLLoadWarning

### ISSUE THREAD

**Issue by sablanchard** [UDST/urbansim#216]
Getting yaml.load() deprecation warning:
```
YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
```

can be fixed with for example:

```
from yaml import CLoader as Loader, CDumper as Dumper

d= yaml.load(f, Loader=Loader)
```

**Comment by cvanoli** [comment]
Another way to solve this is to replace `yaml.load()` by `yaml.safe_load()`

**Comment by smmaurer** [comment]
Same problem as https://github.com/UDST/zone_model/issues/17, more info at https://msg.pyyaml.org/load.

Looks like replacing `yaml.load()` with `yaml.full_load()` will definitely not break anything, because that's the current default loader.

Using `yaml.safe_load()` would probably work too, because as far as i can tell we're only using yaml to store Python dicts and lists ([urbansim/utils/yamlio.py](https://github.com/UDST/urbansim/blob/master/urbansim/utils/yamlio.py)). 

The only reference i can find for the different between "safe" and "full" loading is in the [PyYaml code](https://github.com/yaml/pyyaml/blob/master/lib3/yaml/constructor.py#L473) itself. It looks like "full" includes support for some python-specific data types like class instances. 

My inclination would be to use `safe_load()` going forward, but perhaps use `full_load()` for updating older code, just to be certain we don't break anything that depends on the urbansim utilities..

**Comment by smmaurer** [comment]
Fixed in PR #224.
